src/styles/pages/_object-detail.scss

```diff
@@ -96,7 +96,7 @@ $metadata-collapsed-width: 36vw;
 		}
 
 		.flowplayer {
-			max-height: calc(100vh - 15.1rem); // 15.1rem: navs height
+			height: calc(100vh - 15rem); // 15rem: navs height
```

Comment:
`nav-height` + `subnav-height` is niet consistent doorheen resoluties, is deze code daarop voorzien?

Reply:
aangepast met variables:
```scss
		.flowplayer {
			height: calc(100vh - $c-navigation-height - $c-navigation-height-contextual);

			@include respond-at("md") {
				margin-top: calc(100vh - $c-navigation-height-md - $c-navigation-height-contextual-md);
			}
		}
```